### PR TITLE
Add tests for load balancer and instance

### PR DESCRIPTION
**Pull request by vishalanarase**
Update client to use clinter interface
Add test for load balancer helper functions
Add test for the load balancer interface methods
Add test for instance interface methods

### CODE REVIEW

**Comment by RealHarshThakur** [comment]
Do we need to vendor the packages?

**Comment by vishalanarase** [comment]
> 

Yes, we need

File: doc/yaml/logs [hunk 1]
```diff
@@ -0,0 +1,65 @@
+I0209 04:43:00.001907       1 main.go:31] Starting ccm with CIVO_API_URL: https://api.civo.com, CIVO_REGION: lon1, CIVO_CLUSTER_ID: 1f49054e-1263-43a7-b7fb-2ea1c0f24d89
```
Comment (RealHarshThakur) [inline doc/yaml/logs]: Maybe add it in git ignore?

Comment (vishalanarase) [reply doc/yaml/logs]: Yes, I removed it in the last commit.

**Review by RealHarshThakur — APPROVED** [review]

**Pull request merged**